class-04-templates/pair-assignment/starter-code/scripts/article.js

```diff
@@ -0,0 +1,48 @@
+// var articles = [];
+//
+// function Article (opts) {
+//   this.author = opts.author;
+//   this.authorUrl = opts.authorUrl;
+//   this.title = opts.title;
+//   this.category = opts.category;
+//   this.body = opts.body;
+//   this.publishedOn = opts.publishedOn;
+// }
+
+Handlebars.registerHelper('pubDifference', function(){
+  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+  this.publishStatus = this.publishedOn ? 'published ' + this.daysAgo + ' days ago' : '(draft)';
+  return this.publishStatus;
+})
+
+
+// Article.prototype.toHtml = function() {
+  // DONE: Use handlebars to render your articles.
```

Comment:
You want to keep this function in the code - so instead of running in an IIFE, just make the code part of the toHtml function